scripts/conda/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "omniscidb-cpu" %}
```

Comment:
I suggest copying the content from https://github.com/conda-forge/omniscidb-feedstock/blob/master/recipe/meta.yaml rather than from the old omniscidb-cpu-feedstock.